pandas/tools/merge.py

```diff
@@ -1283,7 +1283,7 @@ def concat(objs, axis=0, join='outer', join_axes=None, ignore_index=False,
         argument, unless it is passed, in which case the values will be
         selected (see below). Any None objects will be dropped silently unless
         they are all None in which case a ValueError will be raised
-    axis : {0, 1, ...}, default 0
+    axis : {0, 1, 'rows', 'columns', ...}, default 0
```

Comment:
Can you make this something like `{0/'index', 1/'columns'}` to make it clear which are equivalent?